translations/search+intl-icu.da.xlf

```diff
@@ -21,6 +17,10 @@
         <source>You can search for case number, address or parts of an address.</source>
         <target>Der kan søges på sagsnummer, adresse eller dele af en adresse.</target>
       </trans-unit>
+      <trans-unit id="HUB_uls" resname="No results for %search%">
+        <source>No results for %search%</source>
+        <target>Søgning på: {searchTerm}, gav ingen resultater.</target>
```

Comment:
The placeholders in the source and target strings should be the same.

Reply:
Fixed in 32582b7d6b8c5be72f347f8f674ae75334c57b7d.